detect_secrets/plugins/keyword.py

```diff
@@ -144,17 +172,20 @@
 }
 # Includes ], ', " as closing
 CLOSING = r'[]\'"]{0,2}'
-DENYLIST_REGEX = r'|'.join(DENYLIST)
+AFFIX_REGEX = r'[a-zA-Z0-9_-]*'
+DENYLIST_REGEX = r'(' + r'|'.join(DENYLIST) + r')({suffix})?'.format(suffix=AFFIX_REGEX)
+DENYLIST_REGEX_WITH_PREV = r'({prefix})('.format(prefix=AFFIX_REGEX) \
```

Comment:
Did you mean `DENYLIST_REGEX_WITH_PREFIX`?

Reply:
Done!